Makefile

```diff
@@ -46,6 +46,11 @@ ifdef GCSSDK_ROOT_DIR
 CMAKE_FLAGS += -DGCSSDK_ROOT_DIR=$(GCSSDK_ROOT_DIR)
 endif
 
+ifdef CONDA_PREFIX
```

Comment:
Is conda prefix an env variable ? Can it be set for other projects - should this be something VELOX_CONDA_PREFIX so that we can be sure that you intend to use conda for the build ?

Reply:
On second thought I agree with you I think it would make sense to have automatic detection of conda with the option to override. I think in most cases if you start building velox within a conda env it was intentional, if not you can disable it...

Reply:
Done